task1/src/handlers/command_handler.py

```diff
@@ -17,7 +18,8 @@ def process_message(self, message):
             Commands.PARTICIPANTS_COUNT: self.handle_participants_count,
             Commands.ROCK_PAPER_SCISSORS: self.handle_rock_paper_scissors,
             Commands.GAME_STEP: lambda: self.handle_game_step(message),
-            Commands.PRIVATE_MESSAGE: lambda: self.handle_private_message(message)
+            Commands.PRIVATE_MESSAGE: lambda: self.handle_private_message(message),
+            Commands.SERVER_TIME: self.handle_server_time_message
         }
```

Comment:
same there, 
should be class constant instead of declaration + initialization on every function call?

Reply:
make sense